src/convertToCase/inputValidation.js

```diff
@@ -0,0 +1,33 @@
+const allCases = ['SNAKE', 'KEBAB', 'UPPER', 'CAMEL', 'PASCAL'];
+
+function inputValidation(originalText, targetCase) {
+  const errors = [];
+
+  if (!originalText) {
+    errors.push({
+      message:
+        // eslint-disable-next-line max-len
+        'Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+    });
+  }
+
+  if (!targetCase) {
```

Comment:
Similar to the previous comment, it would be beneficial to check if `targetCase` is a string to prevent potential errors.